src/components/Search/FilterDropdowns/UserSelectPopup.tsx

```diff
@@ -91,7 +103,7 @@ function UserSelectPopup({value, closeOverlay, onChange}: UserSelectPopupProps)
         const personalDetailList = filteredOptions.personalDetails
             .map((participant) => ({
                 ...participant,
-                isSelected: selectedOptions.some((selectedOption) => selectedOption.accountID === participant.accountID),
+                isSelected: selectedAccountIDs.has(participant.accountID),
```

Comment:
In this useMemo you have also the same warning about the `selectedOptions`

Do you need the `selectedOptions` here? Can you leave a comment explaining that in the code if so please